HomeWork/Aliments.js

```diff
@@ -0,0 +1,23 @@
+import { Products } from "./Products.js";
+export class Aliments extends Products {
+    
+    #expirationDate;
+    #grammage; 
+
+    constructor({ name, price, expirationDate,  grammage}) {
+        super({ name, price});
+        this.#expirationDate = expirationDate;
+        this.#grammage = grammage;
+    }
+
+    get expirationDate() {
+        return this.#expirationDate;
+    }
+
+    print() {
+        console.log(super.name);
```

Comment:
The getters are 'inherited' from the parent class.

If you log `Object.getOwnPropertyNames(Object.getPrototypeOf(this))` you will find `expirationDate` getter among the methods of this class.

If you log `Object.getOwnPropertyNames(Object.getPrototypeOf(Object.getPrototypeOf(this)))` you will find `price` and `name` getters among the methods of the parent class.

A getter is nothing else than a property defined with some restrictions in its descriptor, the most important being that it's read-only. So when you access that property on an object the prototypal chain is searched for it if it's not found among object's own properties.